### DISCUSSION TITLE
Problem with Summary / Summaries

### DISCUSSION THREAD

**Discussion by ARa2of** [Hugovdberg/PIconnect#638] [Q&A] [answered]
Hello Hugo and All, 

Thanks for the great work.

I am having a problem with the Extracting summaries. Usually, this problem occurs whenever I use a module that has an option to add the summary_types (i.e., filtered_summaries, summaries, and summary), here is a screenshot of the message appears: 

PythonException: since Python.NET 3.0 int can not be converted to Enum implicitly. Use Enum(int_value)

![image](https://user-images.githubusercontent.com/69669859/171002963-f873179d-cdd3-41b5-81fe-7d07e83fb648.png)
![image](https://user-images.githubusercontent.com/69669859/171003056-3f9b9fab-6dd5-4b1a-8f93-7791ede74541.png)

interpolated_values module works fine but it doesn't has the option to select the summary type. Also, how to reproduce the results obtained using the interpolated_values on Excel PIDataLInk? 

My python version is Python 3.9.7. 

I'd appreciate your help with this. 

Thank you. 
Ahmed

**Comment by ARa2of** [comment, marked as answer]
I have managed to solve the problem. By following this old [issue](https://github.com/Hugovdberg/PIconnect/issues/630), directed me to the [stackoverflow discussion ](https://stackoverflow.com/questions/67418533/how-to-fix-error-during-pythonnet-installation), and then installed the correct [Python.NET](https://pypi.org/project/pythonnet/) version for windows 64-bit (pythonnet‑2.5.2‑cp39‑cp39‑win_amd64.whl) from [here](https://www.lfd.uci.edu/~gohlke/pythonlibs/#pythonnet).   

Anyway, thanks Hugo and all others for developing this amazing Package. I was actually struggling with the excel PIDataLink. But still would appreciate knowing how to reproduce the results obtained using the interpolated_values on Excel PIDataLInk? 

Thank you

**Reply by Hugovdberg** [reply]
The PI DataLink equivalent of `interpolated_values` is "Sampled Data" or `PISampDat`.
